tools/ci.py

```diff
@@ -196,9 +196,14 @@ def runner_types(ctx: Context, event_name: str):
 
     # This is a push or a scheduled event
     ctx.info(f"Running from a {event_name!r} event")
+    
```

Comment:
This change should no longer be required with the environmental variables we setup on your fork.

Reply:
un did code change